diff --git a/CMakeLists.txt b/CMakeLists.txt
index 77b765c3..bcf7bf02 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,6 +3,7 @@
 cmake_minimum_required(VERSION 3.0.2)
 
 if(APPLE)
+    SET(CMAKE_OSX_ARCHITECTURES "arm64" CACHE STRING "Build architectures for Mac OS X" FORCE)
     # The workaround for try_compile failing with code signing
     # since cmake-3.18.2, not required
     set(CMAKE_TRY_COMPILE_PLATFORM_VARIABLES
